python/tvm/relay/frontend/pytorch.py

```diff
@@ -2938,7 +2948,7 @@ def create_convert_map(self):
             "aten::pixel_shuffle": self.pixel_shuffle,
             "aten::device": self.none,
             "prim::device": self.none,
-            "aten::sub": self.make_elemwise("subtract"),
+            "aten::sub": self.sub,
```

Comment:
It seems this change breaks `test_lstm.py`:
```
FAILED test_lstm.py::test_custom_lstm - AttributeError: 'function' object has no attribute 'dtype'
```

Reply:
I'll see to fixing it. Thank you.